index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>My Portfolio</title>
+</head>
+<body> 
+ <div class="navmenu">
+    <header class="logo">My logo</header>
+    <nav class="menu">
+    <div class="button">
+        <img src="Union.png" alt="hamburger">
+    </div>
+
+    </nav>
+</div>
+<div class="shapes">
+    <img src="header-shapesmobile@2x.png" alt="background-image">
+</div>
+<div class="maintext">
+<h1 class="name">I'm Nabil Glad to see you!</h1>
+<p class="intro">I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+<p class="touch">LET'S CONNECT</p>
+<img class="icon" src="socialmediaicons.png" alt="socialmediaicon">
```

Comment:
- [ ] Each social icon should have a link. I would recommend adding separate icons with links (You can use `a` tag for that). 
- remember to use a `ul` tag to wrap all the social icons and use `li` tag to wrap each of the social icons. This is because the social icons are unordered lists and therefore should be wrapped with a ul and li tag and also to improve the code structure.

Reply:
- [ ] [REQUIRED] Kindly ensure that each social icon has a link. As recommended by the previous reviewer. Also ensure to use a `ul` tag to wrap all the social icons and use `li` tag to wrap each of the social icons. This is because the social icons are unordered lists and therefore should be wrapped with a `ul` and `li` tag and also to improve the code structure.